app.js

```diff
@@ -0,0 +1,127 @@
+// some jquery for my animation
+$('.slide-box').bind('animationend webkitAnimationEnd oAnimationEnd MSAnimationEnd', function(e) { $(this).remove(); });
+
+
+
+// Initialize Firebase
+// Initialize Firebase
+var config = {
+  apiKey: "AIzaSyBPzbEOkfWd2QX6pDuULJFsOpNSkDItWXo",
+  authDomain: "train-activity-c6ac5.firebaseapp.com",
+  databaseURL: "https://train-activity-c6ac5.firebaseio.com",
+  projectId: "train-activity-c6ac5",
+  storageBucket: "",
+  messagingSenderId: "759617035295"
+};
+
+firebase.initializeApp(config);
+
+// Create a variable to reference the database.
+var database = firebase.database();
+
+
+
+// Capture Button Click
+$("button").on("click", function(event) {
+
+  event.preventDefault();
+
+  // Grabbed values from text boxes
+  var trainName = $("#train-input").val().trim();
+  var destination = $("#destination-input").val().trim();
+  var firstTime = $("#fistTime-input").val().trim();
+  var frequency = $("#frequency-input").val().trim();
+
+  // Code for handling the push
+  database.ref().push({
+
+    trainName: trainName,
+    destination: destination,
+    firstTime: firstTime,
+    frequency: frequency
+
+    // dateAdded: firebase.database.ServerValue.TIMESTAMP
+  });
+
+  console.log(trainName);
+  console.log(destination);
+  console.log(firstTime);
+  console.log(frequency);
```

Comment:
I would just comma separate these values, log functions like this are process heavy so in this situation you just need the values it is better practice to just use one console.log and pass each variable to them and comma separate them.